src/plugins/data/public/query/timefilter/timefilter.ts

```diff
@@ -109,6 +114,10 @@ export class Timefilter {
     };
   };
 
+  public getAbsoluteTime() {
```

Comment:
I don't fully understand why we need both `getAbsoluteTime` and `getForceNow`?

Reply:
* `getAbsoluteTime` is same as `getTime` but it also will convert relative time range to absolute time range
* `getForceNow` is internal and private and can be replaced with `this.nowProvider.get()`, I'll clean this up

Reply:
I removed `getForceNow`